### PR TITLE
Log request_id from x-request-id header

### PR DESCRIPTION
**Pull request by jairhenrique**

### CODE REVIEW

**Comment by jrobichaud** [comment]
Thank you for your pull request, I will look into it tonight.

I think we should add as well `X-Correlation-ID` as `correlation_id` when in the request, do you agree?

File: django_structlog/middlewares/request.py [hunk 1]
```diff
@@ -9,6 +9,13 @@
 logger = structlog.getLogger(__name__)
 
 
+def get_x_request_id(request):  # pragma: no cover
+    if hasattr(request, "headers"):
+        return request.headers.get("x-request-id")
+
+    return request.META.get("HTTP_X_REQUEST_ID")
```
Comment (jrobichaud) [inline django_structlog/middlewares/request.py]: Can you explain why there is no coverage for this line and why it is there?

Comment (jrobichaud) [reply django_structlog/middlewares/request.py]: Ok I get it, it is a recent change in django 2.2

**Comment by jrobichaud** [comment]
Your change is available in version 1.4.0.

**Comment by jrobichaud** [comment]
Thank you again.

**Comment by jairhenrique** [comment]
🚀

**Pull request merged**